Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -947,7 +971,16 @@ - (void)lt_maybeRollLogFileDueToSize {
     // We specifically wrote our own getter/setter method to allow us to do this (for performance reasons).
 
     if (_maximumFileSize > 0) {
-        unsigned long long fileSize = [_currentLogFileHandle offsetInFile];
+        unsigned long long fileSize;
+        if (@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)) {
+            __autoreleasing NSError *error;
+            BOOL succeed = [_currentLogFileHandle getOffset:&fileSize error:&error];
+            if (!succeed) {
+                NSLogError(@"DDFileLogger: Failed to get offset: %@", error);
```

Comment:
I wonder if we should abort (return) here. Or at least explicitly set the `fileSize` to 0. Would be interesting what the old `-offsetInFile` returned in case of failure.

Reply:
Ok, just checked. `-offsetInFile` raised an exception. So I'm really in favor of adding a `return` here.

Reply:
In case of failure, fileSize is 0 in default.

But `return` is a good idea!

Reply:
`return` Done!